configure.ac

```diff
@@ -335,6 +335,19 @@ AS_IF([test -n "$with_systemduserunitdir" -a "x$with_systemduserunitdir" != xno]
 AM_CONDITIONAL([ENABLE_SYSTEMD], [test "x$enable_systemd" != xno])
 
 
+# Check for Ubuntu icons
+AC_MSG_CHECKING([whether to enable Upstart job(s)])
+AC_ARG_ENABLE([upstart], [AC_HELP_STRING([--enable-upstart],
+        [enable Upstart job(s)])],
+        [enable_upstart=$enableval],[enable_upstart=no])
+AS_IF([test "x$enable_upstart" != xno], [
+        AC_MSG_RESULT([yes])
```

Comment:
Add `enable_upstart=yes` here so the status printed at the end is nice and consistent (otherwise `enable_upstart` may be set to a weird value).